app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -63,15 +63,116 @@ $top-border: solid 1px govuk-colour("mid-grey", $legacy: "grey-3");
   padding: 3px 3px 0 0;
 }
 
+.gem-c-step-nav____title-text-focus {
+  margin-bottom: 13px;
+  display: inline-block;
+}
+
+// Create Chevron icon aligned with text
+.gem-c-step-nav__chevron {
```

Comment:
It seems like this PR adds a lot of CSS to change the chevron from an SVG to CSS. Is this worth the effort when CSS might look different on different browsers but we can be much more sure the SVG won't?

Reply:
I think would prefer to use SVG however, I think some of the reasons stemmed from IE8+ support, this more closely aligned with providing that icon and then enhancing.  (However as noted below this needs an update)

Also it was around reflow requirements namely `Firefox > Zoom Text Only` support (see image below).  It didn't scale or allow the same level of control.  

![image](https://user-images.githubusercontent.com/71266765/152142729-48fcdd2c-3b5f-4a5a-881d-bc99318f5aed.png)

I'd actually prefer this icon to all be shared (Accordion, Step-by-step, + (upcoming update to) [Details](https://components.publishing.service.gov.uk/component-guide/details)) 🤔 - let me revisit

Reply:
Personally considering the small scale of IE8 usage and that the icon isn't essential (the text already conveys the meaning) I'd be happy if there wasn't an icon on IE8.

That's a good point about Firefox but again while it looks a little strange I don't think the user is missing anything since the text already conveys the meaning.

Reply:
> I'd be happy if there wasn't an icon on IE8.

Agreed, happy to drop.

> while it looks a little strange I don't think the user is missing anything since the text already conveys the meaning.

I used to edge on this side of thinking too however as we've had [similar-ish issues](https://github.com/alphagov/govuk_publishing_components/pull/1799) (& [this](https://github.com/alphagov/govuk_publishing_components/pull/1793)) noted as WCAG concerns it I felt it made sense to factor this in for these updates